gateway/api/rest/pom.xml

```diff
@@ -33,17 +33,6 @@
       <artifactId>swagger-annotations</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.swagger</groupId>
```

Comment:
I'm not sure about this one - I think it may be used downstream in f8.  @KurtStam - are you using this somewhere?  I *think* you added it originally, no?